content/references/xrpljs2-migration-guide.jp.md

```diff
@@ -0,0 +1,344 @@
+(async function() {
+  const client = new xrpl.Client('wss://xrplcluster.com');
+
+  await client.connect();
+
+  // Your code here
+
+  client.disconnect();
+})();
+```
+
+
+## バリデーション結果
+
+デフォルトでは、ripple-lib 1.xにおけるほとんどのメソッドは、[コンセンサスプロセス](consensus.html)によって検証された最終結果をリターンするのみでした。xrpl.jsと同等の多くのメソッドは、WebSocket APIをコールするために[`Client.request()`メソッド](https://js.xrpl.org/classes/Client.html#request)を使用します。WebSocket APIにおいて、XRP Ledgerサーバーのデフォルト設定では、検証済みデータだけはなく未検証のデータを含むことがあります。 
+
+[分散型取引所](decentralized-exchange.html)の状態を調べる時のように、完了見込みの多数のトランザクション結果が保留中であるため、現時点のオープンレジャーを使用したい場合があります。また、完了したトランザクション結果を取り込んだ検証済みのレジャーを使用したい場合もあります。
+
+When making API requests with xrpl.js 2.0 using `Client.request()`, you should explicitly [specify what ledger to use](basic-data-types.html#specifying-ledgers). For example, to look up trust lines using the latest _validated ledger_:
+
+xrpl.js 2.0が`Client.request()`を使用してAPIリクエストをする際、明確に[使用するレジャー番号を指定する](basic-data-types.html#specifying-ledgers)必要があります。例えば、最新の_検証済みレジャー_を使用してトラストラインを調べるためには:
+
+**ripple-lib 1.x:**
+
+```js
+const trustlines = await api.getTrustlines("rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn")
+console.log(trustlines)
+```
+
+**xrpl.js 2.0:**
+
+```js
+const trustlines = await client.request({
+  "command": "account_lines",
+  "account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+  "ledger_index": "validated"
+})
+console.log(trustlines.result)
+```
+
+
+## トランザクションの送信
+
+xrpl.jsには、トランザクションの署名および送信のための、また、XRP Ledgerブロックチェーンのトランザクション最終結果の確認を待機するための特有の補助機能があります:
+
+- トランザクション送信および[最終結果](finality-of-results.html)の待機のために`submitAndWait()`を使用します。トランザクションが検証された場合、これは[txメソッド][]レスポンスにリゾルブし、そうでない場合、例外処理(exception)となります。例外処理(exception)は、トランザクションが検証されなかったことを保証しません。例えば、サーバーに[より大きなギャップ](reliable-transaction-submission.html#ledger-gaps)がある場合、トランザクションは、そのギャップの中で検証される可能性があります。
+- 即時の送信およびリターンのために`submit()`を使用します。これは[submitメソッド][]レスポンスにリゾルブし、仮の(最終ではない)結果を表示します。もしXRP Ledgerサーバーへのトランザクション送信に問題があった場合、このメソッドは例外処理(exception)のみとなります。
+
+どちらのメソッドに関しても、準備済みトランザクション説明と[`Wallet`インスタンス](#keys-and-wallets)をパスすることによって、署名済みトランザクションをメソッドに直接パス、もしくは、送信直前にトランザクションに署名することができます。
+
+```js
+const tx_json = await client.autofill({
+  "TransactionType": "AccountSet",
+  "Account": wallet.address, // "wallet"はWalletクラスのインスタンス
+  "SetFlag": xrpl.AccountSetAsfFlags.asfRequireDest
+})
+try {
+  const submit_result = await client.submitAndWait(tx_json, wallet)
+  // submitAndWait() はトランザクションの結果が確定するまでreturnしません。
+  // トランザクションがネットワークに確認されなかった場合、XrplErrorが発生します。
+  // ディザスタリカバリには対応しません。
+  console.log("Transaction result:", submit_result)
+} catch(err) {
+  console.log("Error submitting transaction:", err)
+}
+```
+
+Alternatively, you can use the `sign` method of a wallet to sign a transaction and then use `submitAndWait(tx_blob)` to submit it. This can be useful for building [reliable transaction submission](reliable-transaction-submission.html) that can recover from power outages and other disasters. (The library does not handle disaster recovery on its own.)
```

Comment:
It looks like the following sentences translate this, so the English can be deleted.